migracja.rst

```diff
@@ -0,0 +1,235 @@
+**Laboratorium EE**
+
+*29 XII 2016*
+
+Model migracji z systemów EZD PUW i eDok
+========================================
+
+Wstęp
+<<<<<
+
+Operacja migracji danych z instalacji istniejącego systemu EZD do innego system EZD, powinna być
+realizowana za pośrednictwem pliku pośredniego (lub pewnej liczby plików) — a nie on-line.
+
+Dane pośrednie będą zapisane w formacie, który jest wspólny dla obu systemów.
+
+Migracja a eksport wybranych danych
+<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+
+Przekazywanie danych z systemu EZD może mieć miejsce w kilku sytuacjach. Każdy system EZD musi mieć
+zaimplementowane operacje eksportu (przekazywania) danych do innych systemów. Przynajmniej:
+
+- przesyłanie dokumentów do innych systemów teleinformatycznych, w szczególności przez eksport
+  dokumentów i ich metadanych z zachowaniem powiązań pomiędzy tymi dokumentami i metadanymi
+  zob. §6 pkt 14 Rozporządzenia Ministra Spraw Wewnętrznych i Administracji z dnia 30 października
+  2006 r. w sprawie szczegółowego sposobu postępowania z dokumentami elektronicznymi
+  http://isap.sejm.gov.pl/DetailsServlet?id=WDU20062061518)
+- wytworzenie „paczki archiwalnej” z dokumentami, do przekazania do Archiwum Państwowego oraz spisu
+  zdawczo-odbiorczego odpowiadającego określonej paczce archiwalnej (zob. §6 pkt 13 14
+  Rozporządzenia MSWiA z dnia 30 października 2006)
+
+Do marca 2017r., jest też realizowany projekt we współpracy MC - COI - NSA - WSA - PUW, mający na celu
```

Comment:
"też" jest - dla mnie - wybijający, bo ciężko wychwycić co on potwierdza. Proponuje: ``Należy zaznaczyć, że do marca 2017r. jest realizowany również projekt we współpracy MC - COI - NSA - WSA - PUW, mający na celu (…)``.  Nie zamierzam się jednak upierać.